pontificia-universidade-catolica-do-parana-abnt.csl

```diff
@@ -222,196 +471,681 @@
+            <text macro="issued-year"/>
+            <text macro="citation-locator" prefix=", "/>
+          </group>
+        </else-if>
+        <else>
+          <!-- do contrário é o formato padrão de autor e data separados por vírgula -->
+          <group>
+            <text macro="author-short" suffix=", "/>
+            <text macro="issued-year"/>
+            <text macro="citation-locator" prefix=", "/>
+          </group>
+        </else>
+      </choose>
     </layout>
   </citation>
-  <bibliography hanging-indent="false" entry-spacing="1">
+  <bibliography et-al-min="4" et-al-use-first="1" line-spacing="1" entry-spacing="1">
+    <!-- REFERÊNCIAS: a partir de 4 autores utiliza et al., alinhamento a esquerda, espaçamento simples e uma linha entre referências -->
     <sort>
-      <key macro="author"/>
-      <key macro="issued-year" sort="descending"/>
+      <key macro="author-short"/>
+      <key variable="year-suffix"/>
+      <key macro="title"/>
+      <!-- ordena primeiro pela macro autor -->
+      <key variable="issued"/>
+      <!-- em seguida pela data -->
     </sort>
     <layout>
-      <group delimiter=". ">
-        <text macro="author"/>
-        <text macro="title"/>
-        <choose>
-          <if type="bill" match="any">
-            <text variable="genre" text-case="uppercase"/>
-            <text variable="number"/>
-            <group delimiter=" ">
-              <text term="section" text-case="capitalize-first"/>
-              <text variable="section"/>
-            </group>
-            <text variable="publisher-place"/>
-            <text macro="issued-legislation"/>
-            <text variable="container-title"/>
-            <text macro="locators"/>
-          </if>
-          <else-if type="book" match="any">
-            <text macro="translator"/>
-            <text macro="edition"/>
-            <text macro="publisher"/>
-            <text macro="issued"/>
-          </else-if>
-          <else-if type="patent" match="any">
-            <group delimiter=": ">
-              <text term="patent" text-case="capitalize-first"/>
-              <text variable="number"/>
-            </group>
-            <text macro="issued"/>
-          </else-if>
-          <else-if type="webpage post post-weblog" match="any">
-            <group delimiter=": ">
-              <text term="in" font-weight="bold" text-case="capitalize-first"/>
-              <text variable="container-title" font-weight="bold"/>
-            </group>
+      <choose>
+        <if type="article">
+          <!-- Preprint (Zotero) e Programa de computador (Mendeley)-->
+          <group>
+            <text macro="author" suffix=". "/>
+            <text variable="authority" suffix=". "/>
+            <text macro="title" suffix=" "/>
+            <text variable="medium" suffix=". "/>
+            <text macro="publisher" suffix=" "/>
+            <text variable="genre" suffix="."/>
+            <text macro="access"/>
```

Comment:
Set all these dots on the group level. Check the rest of yur style for this issue. It is repeated.